README.md

```diff
@@ -3,6 +3,85 @@ JupyterLab Extension Builder
 
 Tools for building JupyterLab extensions.
 
+An extension to JupyterLab is a module that provides one or more
+plugins to the JupyterLab application.  This library provides
+a build script for generating third party extension JavaScript
+bundles.  
+
+Simple extensions can be created by using the `buildExtension` function
+with the default options.  More advanced extensions may require additional
+configuration.
+
+A simple extension entry point could look like: 
+
+```javascript
+module.exports = [{
+    id: 'my-cool-extension',
+    activate: function(app) {
+       console.log(app.commands);
+    }
+}];
+```
+
+The above could be built using the following script:
+
+```javascript
+var buildExtension = require('jupyterlab-extension-builder').buildExtension;
+
+buildExtension({
+    name: 'my-cool-extension',
+    entryPath: './index.js',
+    outputPath: './build'
+});
+```
+
+In this case the builder script will create the following files:
+
+```
+build/my-cool-extension.js
+build/my-cool-extension.js.manifest
+```
+
+Other extensions may produce other files in the build directory
+depending on the complexity of extension.  The two files above
+are used by JupyterLab server to determine the entry point file(s) and 
```

Comment:
...by the JupyterLab...

Not sure whether it should be entry point or entrypoint. Either I suspect is fine as long as we are consistent.